lib/src/request/CommonMediaRequest.ts

```diff
@@ -12,46 +13,51 @@ export type CommonMediaRequest = {
 	/**
 	 * The URL of the request.
 	 */
-	url: string
+	url: string;
 
 	/**
 	 * The request's method (GET, POST, etc).
 	 */
-	method: string
+	method?: string;
+
+	/**
+	 * The body of the request.
+	 */
+	body?: BodyInit;
```

Comment:
@agajassi Will this work for your DRM changes?

Reply:
Yes. Thanks. That is exactly what I needed.